reference/5.1/Microsoft.PowerShell.Core/About/about_Split.md

```diff
@@ -107,52 +104,66 @@ FirstName
 Address
 ```
 
-In the following example, \<Max-substrings\> is set to 3. This results in
-three splits of the string values, but a total of five strings in the
-resulting output; the delimiter is included after the splits, until the
-maximum of three substrings is reached. Additional delimiters in the final
-substring become part of the substring.
-
-```powershell
-'Chocolate-Vanilla-Strawberry-Blueberry' -split '(-)', 3
-```
+### `<Max-substrings>`
 
-```output
-Chocolate
--
-Vanilla
--
-Strawberry-Blueberry
-```
-
-### \<Max-substrings\>
-
-Specifies the maximum number of times that a string is split. The default is
-all the substrings split by the delimiter. If there are more substrings, they
-are concatenated to the final substring. If there are fewer substrings, all
-the substrings are returned. A value of 0 and negative values return all the
+Specifies the maximum number of substrings returned by the split operation. The
+default is all the substrings split by the delimiter. If there are more
+substrings, they are concatenated to the final substring. If there are fewer
+substrings, all the substrings are returned. A value of 0 returns all the
```

Comment:
I could be wrong on these two, the "the" just reads as unnecessary to me. Feel free to ignore on both if I'm crazy
```suggestion
substrings, all substrings are returned. A value of 0 returns all the
```